content/docs/concepts/traffic-management/network/index.md

```diff
@@ -0,0 +1,320 @@
+of incorrect configurations include incompatible or restrictive timeouts across
+service calls.
+
+With Istio, you can use application-layer fault injection instead of killing
+pods, delaying packets, or corrupting packets at the TCP layer. You can inject
+more relevant failures at the application layer, such as HTTP error codes, to
+test the resilience of an application.
+
+You can inject faults into requests that match specific conditions, and you can
+restrict the percentage of requests Istio subjects to faults.
+
+You can inject two types of faults:
+
+-  **Delays:** Delays are timing failures. They mimic increased network latency
+   or an overloaded upstream service.
+
+-  **Aborts:** Aborts are crash failures. They mimic failures in upstream
+   services. Aborts usually manifest in the form of HTTP error codes or TCP
+   connection failures.
+
+You can configure a virtual service to inject one or more faults while
+forwarding HTTP requests to the rule's corresponding request destination. The
+faults can be either delays or aborts.
+
+### Introduce a 5 second delay in 10% of requests
+
+You can configure a virtual service to introduce a 5 second delay for 10% of
+the requests to the `ratings` service.
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: ratings
+spec:
+  hosts:
+  - ratings
+  http:
+  - fault:
+      delay:
+        percentage:
+  value: 0.1
+        fixedDelay: 5s
+    route:
+    - destination:
+        host: ratings
+        subset: v1
+{{< /text >}}
+
+### Return an HTTP 400 error code for 10% of requests
+
+You can configure an abort instead to terminate a request and simulate a
+failure.
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: ratings
+spec:
+  hosts:
+  - ratings
+  http:
+  - fault:
+      abort:
+        percentage:
+          value: 0.1
+        httpStatus: 400
+    route:
+    - destination:
+        host: ratings
+        subset: v1
+{{< /text >}}
+
+### Combine delay and abort faults
+
+You can use delay and abort faults together. The following configuration
+introduces a delay of 5 seconds for all requests from the `v2` subset of the
+`ratings` service to the `v1` subset of the `ratings` service and an abort for
+10% of them:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: ratings
+spec:
+  hosts:
+  - ratings
+  http:
+  - match:
+    - sourceLabels:
+        app: reviews
+        version: v2
+    fault:
+      delay:
+        fixedDelay: 5s
+      abort:
+        percentage:
+          value: 0.1
+        httpStatus: 400
+    route:
+    - destination:
+        host: ratings
+        subset: v1
+{{< /text >}}
+
+For detailed instructions on how to configure delays and aborts, visit our
+[fault injection task](/docs/tasks/traffic-management/fault-injection/).
+
+## Compatibility with fault tolerance libraries
```

Comment:
I would remove this section completely. its no longer relevant. It was 3 years ago but not now.

Reply:
@frankbu @geeknoid @andraxylia Do you concur? Relevance is a tricky thing and I don't have insight into whether this is important to our readers.

Reply:
I think this was added when we were trying to get people to move their applications to Istio from what was then the dominant framework for microservices-based apps: NetflixOSS/Hystrix. I'm guessing that mixing frameworks isn't much of a requirement anymore.

Reply:
@frankbu @rshriram This section helps clarify how Istio interacts with the application-level libraries and that is information readers need. In a future effort, we can move the section to a different page, but, for now, I think we should keep it here.